client/src/app.config.router.js

```diff
@@ -4,6 +4,7 @@ const homeItemComponent = require('./states/student/home/component');
 const studentLecturesListComponent = require('./states/student/lecturesList/component');
 const studentComponent = require('./states/student/component');
 const projectComponent = require('./states/student/project/component');
+const editComponent = require('./states/student/edit/component');
```

Comment:
if it possible I will fix it in next commit - when I finish "edit user". Now server 500 error(